Api/CreditmemoByIncrementIdInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace SnowIO\ExtendedSalesRepositories\Api;
+
+/**
+ * Class CreditmemoByIncrementIdInterface
+ *
+ * @api
+ */
+interface CreditmemoByIncrementIdInterface extends \Magento\Framework\Api\ExtensibleDataInterface
```

Comment:
Preference for`CreditmemoOrderByIncrementIdInterface` to follow the pattern of `ShipOrderByIncrementIdInterface` where we describe the interface as the intent for the order and the mechanism by which we will identify the order.
Affects the class name of the implementation too to be consistent.

This is just my personal preference.

Reply:
yeh nice, alex suggested CreditmemoByOrderIncrementIdInterface which is good for me @thisisandrew